casper/src/main/scala/coop/rchain/casper/genesis/contracts/StandardDeploys.scala

```diff
@@ -107,4 +110,12 @@ object StandardDeploys {
       "be8ae000e4d2f29c73f792705314d71b6c0d56d7c640c6b4df9fabf90518c623",
       1551879405043L
     )
+
+  def testRev(genesisPk: PublicKey, vaults: Seq[Vault], supply: Long): DeployData =
+    toDeploy(
+      RevGenerator(RevAddress.fromPublicKey(genesisPk).get, vaults, supply),
+      s"${Base16.encode(genesisPk.bytes)}",
```

Comment:
a `.toBase16` method on PrivateKey and PublicKey would help me a lot when reading such code, and probably would be handy when debugging (easier logging / setting up a watch)